cockatrice/src/remoteclient.cpp

```diff
@@ -299,6 +343,30 @@ void RemoteClient::doRegisterToServer(const QString &hostname, unsigned int port
     setStatus(StatusRegistering);
 }
 
+void RemoteClient::doForgotPassword(const QString &hostname, unsigned int port, const QString &_userName, const QString &_email)
+{
+	doDisconnectFromServer();
+
+	userName = _userName;
+	email = _email;
```

Comment:
I would agree we should be checking email, but I was over-ruled.  (see the discussion above) . However , the email can be still be verified based on the configuration of the server operator (along with other checks).  In the default configuration of how the system is configured email verification was suggested to not happen.